tasks/framework-tools/yarn.lock

```diff
@@ -49,7 +49,7 @@
   resolved "https://registry.yarnpkg.com/@babel/compat-data/-/compat-data-7.14.7.tgz#7b047d7a3a89a67d2258dc61f604f098f1bc7e08"
   integrity sha512-nS6dZaISCXJ3+518CWiBfEr//gHyMO02uDxBkXTKZDN5POruCnOZ1N4YBRZDCabwF8nZMWBpRxIicmXtBs+fvw==
 
-"@babel/core@7.14.6":
+"@babel/core@7.14.6", "@babel/core@^7.7.2":
```

Comment:
I wonder why this older version got added

Reply:
Remember our old friend `yarn why`

Reply:
Oh, for sure I why'd 😉